<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeaksWorkspaceWidget</class>
 <widget class="QWidget" name="PeaksWorkspaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,1">
   <item>
    <layout class="QHBoxLayout" name="infoLayout">
     <item>
      <widget class="QLabel" name="lblWorkspaceName">
       <property name="text">
        <string>[Workspace]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblWorkspaceState">
       <property name="text">
        <string>[Integrated/UnIntegrated]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblWorkspaceCoordinates">
       <property name="text">
        <string>[Viewing Coordinate System]</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ckExpand">
       <property name="text">
        <string>Expand</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tableLayout">
     <item>
      <widget class="QTableView" name="tblPeaks">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlsLayout">
     <item>
      <widget class="QPushButton" name="btnHide">
       <property name="text">
        <string>Hide</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Peak Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPeakColor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Peak Background Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBackgroundColor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckShowBackground">
       <property name="text">
        <string>Show Background</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
